packages/dotnet/ReadMe/HarJsonObjectModels/Content.cs

```diff
@@ -2,7 +2,6 @@
 
 namespace ReadMe.HarJsonObjectModels
 {
-  [JsonObject(ItemNullValueHandling = NullValueHandling.Ignore)]
```

Comment:
What's this for?

Reply:
This means that if any item in this class is `null` that when we convert it to JSON it'll be removed from the resulting JSON.